samples/maps/demo/flowmap-flight-routes/demo.js

```diff
@@ -0,0 +1,197 @@
+(async () => {
+    const mapData = await fetch(
+        'https://code.highcharts.com/mapdata/countries/gb/gb-all.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+
+        title: {
+            text: 'Highmaps basic flight routes demo'
+        },
+
+        subtitle: {
+            text: 'Demo of Highcharts map with flowmap'
+        },
+
+        mapNavigation: {
+            enabled: true
+        },
+
+        tooltip: {
+            formatter: function () {
```

Comment:
Use `headerFormat` and `pointFormat` instead. Make one for each series if checking for `this.point.lat` is a problem.